compose/gettingstarted.md

```diff
@@ -32,7 +32,7 @@ Define the application dependencies.
         from flask import Flask
 
         app = Flask(__name__)
-        cache = redis.Redis(host='redis', port=6379)
+        cache = redis.Redis(host='redis', port=5000)
```

Comment:
This is the port for the redis container, which is 6379 by default, so I don't think this is correct

Reply:
My Mistake